app/backend/recipes/src/main/java/com/juotava/recipes/service/RecipesService.java

```diff
@@ -109,6 +121,65 @@ public void setCurrentUserToRecipe(Recipe recipe, String auth0id){
         recipe.setCreatedBy(auth0id);
     }
 
+    public List<RecipeList> getRecipeListsByUser(String auth0id) {
+        return this.recipeListRepository.findByCreatedByAuth0id(auth0id);
+    }
+
+    public UUID createRecipeList(String title, String auth0id) {
```

Comment:
Might be good to use try/catch here as sending a title with more than 255 characters leads to the list not being created in the db -> uncaught exception.
I'd rather the code does not crash but return "false" to the user